twokenize.py

```diff
@@ -285,15 +293,22 @@ def tokenize(text):
 # We also first unescape &amp;'s, in case the text has been buggily double-escaped.
 def normalizeTextForTagger(text):
     text = text.replace("&amp;", "&")
-    text = HTMLParser.HTMLParser().unescape(text)
+    #http://stackoverflow.com/questions/2360598
+    #/how-do-i-unescape-html-entities-in-a-string-in-python-3-1
+    if sys.version.info[0] > 2:
+        if sys.version.info[0] == 3 and sys.version.info[1] <= 3:
+            text = HTMLParser().unescape(text)
+        else:
+            text = html.unescape(text)
+    else:
+        text = HTMLParser().unescape(text)
     return text
 
 # This is intended for raw tweet text -- we do some HTML entity unescaping before running the tagger.
-# 
+#
 # This function normalizes the input text BEFORE calling the tokenizer.
 # So the tokens you get back may not exactly correspond to
 # substrings of the original text.
 def tokenizeRawTweetText(text):
```

Comment:
Both before and after your changes, this code works poorly on non-unicode text. What do you think about adding `assert isinstance(text, unicode)` to the top of this function and also to the top of the `tokenize` function?